implementer-guide/setup/upgrade.md

```diff
@@ -11,22 +11,45 @@ position: 205
-The only configuration files in the installation folder that will be preserved after the upgrade are _/Telerik.ReportServer.Web/ReportServerAdmin.config_, _/Telerik.ReportServer.Web/TelerikReporting.config_, and _/Services/ServiceAgent.config_.
-This way the previously configured assets storage will be preserved after the upgrade as well as any report generation engine configuration options applied in the _/Telerik.ReportServer.Web/TelerikReporting.config_ file.
 
-The installer has a step that [backs up the assets' storage]({%slug storage-backup%}) used from the Report Server. This step is optional and turned on by default. We recommend not skipping it. On the first run after the upgrade, the Report Server will upgrade the assets storage schema version if necessary. Keep the backed-up assets' storage at least until the Report Server is up and running again.
+Telerik Report Server installer replaces all files in the selected installation folder deployed by the previous installer, including the configuration files `/Telerik.ReportServer.Web/Web.config` and `/Services/Telerik.ReportServer.ServiceAgent.exe.config`.
 
-# Before upgrading the Report Server
+The only configuration files in the installation folder that will be preserved after the upgrade are `/Telerik.ReportServer.Web/ReportServerAdmin.config`, `/Telerik.ReportServer.Web/TelerikReporting.config`, and `/Services/ServiceAgent.config`.
 
-We advise performing a file backup before upgrading. This is specifically recommended if you have changed the application configuration files _/Telerik.ReportServer.Web/Web.config_ and _/Services/Telerik.ReportServer.ServiceAgent.exe.config_
+This way the previously configured assets storage will be preserved after the upgrade as well as any report generation engine configuration options applied in the `/Telerik.ReportServer.Web/TelerikReporting.config` file.
 
-Note the _Telerik Report Server_ settings applied in the IIS console, including the Bindings. You will need to reapply these after the upgrade. 
+The installer has a step that [backs up the assets' storage]({%slug storage-backup%}) used from the Report Server. This step is optional and turned on by default. We recommend keeping it. On the first run after the upgrade, the Report Server will upgrade the assets storage schema version if necessary. Keep the backed-up assets' storage at least until the Report Server is up and running again.
 
-# After upgrading the Report Server
+## Before upgrading the Report Server
+
+We advise performing a file backup before upgrading. This is specifically recommended if you have changed the application configuration files `/Telerik.ReportServer.Web/Web.config` and `/Services/Telerik.ReportServer.ServiceAgent.exe.config`
+
+### IIS Settings
+
+The _Telerik Report Server_ settings applied in the IIS console, including the HTTPS bindings and AppPool identity will be preserved if you are with version `9.2.23.1115 or later` and upgrade to a newer version.
```

Comment:
"...and AppPool identity will be preserved..."
I think there should be a comma between "identity" and "will be":

Reply:
fixed